### ISSUE TITLE
Unit test fails after SCS 3.0 → 3.2 upgrade

### ISSUE THREAD

**Issue by stephane-caron** [stephane-caron/qpsolvers#56]
After pushing some doc commits today I noticed the CI fails:
```
  FAIL: test_sparse_bounds_scs (tests.test_solve_qp.TestSolveQP)
  ----------------------------------------------------------------------
  Traceback (most recent call last):
    File "/home/runner/work/qpsolvers/qpsolvers/tests/test_solve_qp.py", line 430, in test
      self.assertLess(norm(x - known_solution), sol_tolerance)
  AssertionError: 1.3095214160904062e-08 not less than 1e-08
```
When I run tests locally with SCS 3.0.1 all tests pass, but after upgrading to 3.2.0 I see the same errors locally.

@bodono FYI, maybe some defaults changed between 3.0 and 3.2?

The absolute drift is not huge, we can probably set laxer precision settings for SCS so that the CI passes again.

**Comment by stephane-caron** [comment]
Tried it out and the change is only for two tests:
* QP: ``test_sparse_bounds_scs``: bump primal solution tolerance from 1e-8 to 1e-7
* LS: ``test_scs``: bump primal solution tolerance from 1e-7 to 2e-7
With these two changes the [CI passes again](https://github.com/stephane-caron/qpsolvers/actions/runs/1933310931).

**Comment by bodono** [comment]
In 3.2.0 we introduced the ability to reuse the SCS workspace and update the `b` and `c` vectors that defined the problem (see [updated python API docs](https://www.cvxgrp.org/scs/api/python.html#python-interface)). This is useful when, for example, you are solving a sequence of related problems like a lasso with different choices of regularization parameter. To do this in a consistent and clean way I removed the dependency on `b` and `c` from the heuristic data preprocessing (aka equilibration aka normalization) that SCS does. This results in slightly different equilibration and therefore slightly different performance. I tested this on a suite of problems and the overall performance difference was pretty negligible. Probably for some of your problems the performance improved slightly and some got slightly worse, which tripped some tests.
